config/db/seed.js

```diff
@@ -235,6 +230,139 @@ function assignUnits() {
     })
     .catch(e => console.log(e));
 }
+function createActivities() {
+  activities = [
```

Comment:
I would remove the json folder to seedData and convert all the files to .js files so that way you can run javascript code in there which is necessary. Add this data into the activities.js file and use the units enumerator to populate the unit value.

Reply:
This would make assign units obsolete so you can just delete it.

Reply:
Added Utils file. Resolving.